### PR TITLE
enh(drawio): map police properties management latest

### PR DESCRIPTION
**Pull request by Yassir-BenBOUBKER**
## The purpose of this RP is
- To allow the manipulation of the text style of the shapes in drawio.
- Remove the features selected in the **picture** because it is not yet supported by the backend.
image

### CODE REVIEW

File: src/main/webapp/js/grapheditor/Format.js [hunk 1]
```diff
@@ -3143,49 +3145,13 @@ TextFormatPanel.prototype.addFont = function(container)
...
-		mxUtils.write(dirOption, mxResources.get(dirs[i]));
-		dirSelect.appendChild(dirOption);
-	}
 
-	stylePanel5.appendChild(dirSelect);
-	
 	if (!graph.isEditing())
 	{
-		container.appendChild(stylePanel4);
+		// container.appendChild(stylePanel4);
```
Comment (bdauria) [inline src/main/webapp/js/grapheditor/Format.js]: Remove comment if not needed

File: src/main/webapp/styles/centreon.css [hunk 2]
```diff
@@ -31,17 +31,21 @@ div.FormatPanel-perimeterPanel {
   display: none;
 }
 
-div.TextFormatPanel-addFont-stylePanel, 
+/* div.TextFormatPanel-addFont-stylePanel, 
 div.TextFormatPanel-LabelPosition,
 div.TextFormatPanel-writingDirection {
   display: none;
```
Comment (bdauria) [inline src/main/webapp/styles/centreon.css]: Same remark

File: src/main/webapp/styles/centreon.css [hunk 3]
```diff
@@ -31,17 +31,21 @@ div.FormatPanel-perimeterPanel {
...
+
+div.TextFormatPanel-addFont-stylePanel-ContentEditing {
   display: none;
 }
 
+/*
+div.geFormatSection-BaseFormatPanel-createRelativeOption-lineheight {
+  display: none;
+} */
+
```
Comment (bdauria) [inline src/main/webapp/styles/centreon.css]: Same here

**Comment by Thebarda** [comment]
Please provide a description to your PR in order to give more context on what your PR is doing

File: src/main/webapp/js/grapheditor/Format.js [hunk 4]
```diff
@@ -3143,50 +3145,12 @@ TextFormatPanel.prototype.addFont = function(container)
...
-	var dirSelect = document.createElement('select');
-	dirSelect.style.position = 'absolute';
-	dirSelect.style.border = '1px solid rgb(160, 160, 160)';
-	dirSelect.style.left = '126px';
-	dirSelect.style.width = '98px';
-	dirSelect.style.borderRadius = '4px';
-	dirSelect.style.marginTop = '-2px';
 
 	// NOTE: For automatic we use the value null since automatic
 	// requires the text to be non formatted and non-wrapped
```
Comment (bdauria) [inline src/main/webapp/js/grapheditor/Format.js]:
```suggestion
```

**Review by Thebarda — COMMENTED** [review]
There are some conflicts to resolve

**Review by Thebarda — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**